jdbc-adapter/virtualschema-jdbc-adapter/src/test/resources/integration/vschema_complex_inner_join.json

```diff
@@ -0,0 +1,240 @@
+{
+  "testSchema" :
+  [
+    "create table t1(id int, val int)",
+    "create table t2(id int, val int)"
+  ],
+  "testCases" :
+  [
+    {
+      "testQuery": "select a.id, b.id, sum(a.val), sum(b.val) from vs.t1 a inner join vs.t2 b on a.id=b.id group by a.id, b.id having sum(a.val) > 0 order by a.id limit 1000",
+      "expectedPushdownRequest":
+      [
+        {
+          "involvedTables" :
+          [
+            {
+              "columns" :
+              [
+                {
+                  "adapterNotes" : "{\"jdbcDataType\":3,\"typeName\":\"DECIMAL\"}",
```

Comment:
I am curious: where does the name `adapterNotes` come from?

Reply:
I think @andrehacker came up with this name.